_maps/RandomRuins/BeachRuins/beach_ancient_ruin.dmm

```diff
@@ -1,343 +1,8597 @@
+	dir = 4
+	},
+/obj/structure/spacevine/weak,
+/turf/open/floor/concrete/slab_4,
+/area/ruin/beach/complex)
+"jg" = (
+/obj/structure/marker_beacon,
+/obj/effect/turf_decal/trimline/opaque/white/filled/warning{
+	dir = 10
+	},
+/obj/structure/cable{
+	icon_state = "1-4"
+	},
+/turf/open/floor/concrete/pavement{
+	light_range = 2
+	},
+/area/ruin/beach/complex)
+"jk" = (
+/obj/effect/turf_decal/trimline/opaque/white/filled/warning,
+/turf/open/floor/concrete/pavement{
+	light_range = 2
+	},
+/area/ruin/beach/complex)
+"ju" = (
+/obj/effect/turf_decal/trimline/opaque/white/filled/warning{
+	dir = 9
+	},
+/turf/open/floor/plasteel/mono,
+/area/ruin/beach/complex)
+"jE" = (
+/obj/structure/spacevine/weak,
+/obj/effect/decal/cleanable/dirt/dust,
+/turf/open/floor/concrete/slab_4,
+/area/ruin/beach/complex)
+"kh" = (
+/obj/effect/turf_decal/trimline/opaque/white/filled/warning{
+	dir = 1
+	},
+/obj/structure/cable{
+	icon_state = "4-8"
+	},
+/turf/open/floor/concrete/pavement{
+	light_range = 2
+	},
+/area/ruin/beach/complex)
+"ks" = (
+/mob/living/simple_animal/hostile/hivebot/strong{
+	name = "mechanoid";
+	environment_smash = 0
+	},
+/turf/open/floor/concrete/slab_4,
+/area/ruin/beach/complex)
+"kK" = (
+/obj/structure/closet/crate/bin,
+/obj/item/reagent_containers/food/snacks/breadslice/moldy,
+/turf/open/floor/concrete/slab_4,
+/area/ruin/beach/complex)
+"kM" = (
+/obj/effect/turf_decal/trimline/opaque/white/filled/warning{
+	dir = 5
+	},
+/obj/structure/spacevine/weak,
+/turf/open/floor/plasteel/mono,
+/area/ruin/beach/complex)
+"kV" = (
+/obj/structure/cable{
+	icon_state = "1-2"
+	},
+/turf/open/floor/plasteel/stairs,
+/area/ruin/beach/complex)
+"kW" = (
+/obj/effect/turf_decal/siding/white{
+	dir = 1
+	},
+/obj/structure/spacevine/weak,
+/turf/open/floor/concrete/slab_3{
+	light_range = 2
+	},
+/area/ruin/beach/complex)
+"la" = (
+/obj/machinery/power/terminal{
+	dir = 8
+	},
+/obj/structure/catwalk/over/plated_catwalk/dark,
+/obj/machinery/power/shieldwallgen/atmos/roundstart{
+	dir = 4;
+	id = "dropshuttle"
+	},
+/obj/machinery/door/poddoor/shutters{
+	id = "doorshuttle"
+	},
+/obj/structure/cable,
+/obj/structure/cable{
+	icon_state = "1-4"
+	},
+/turf/open/floor/plating,
+/area/ruin/beach/complex/shuttle)
+"li" = (
+/obj/structure/cable{
+	icon_state = "1-2"
+	},
+/obj/structure/spacevine/weak,
+/turf/open/floor/plasteel/mono,
+/area/ruin/beach/complex)
+"lk" = (
+/obj/structure/flora/ausbushes/sunnybush,
+/turf/open/floor/plating/grass{
+	light_range = 2
+	},
+/area/ruin/beach/complex)
+"lw" = (
+/obj/effect/turf_decal/weather/sand{
+	dir = 10
+	},
+/turf/open/water/beach,
+/area/ruin/unpowered)
+"lR" = (
+/obj/machinery/light/small,
+/obj/structure/spacevine/weak,
+/turf/open/floor/concrete/slab_1,
+/area/ruin/beach/complex/wall)
+"lS" = (
+/turf/closed/wall/concrete{
+	desc = "As you draw near the ancient wall, a sense of foreboding overcomes you. You aren't sure why, but you feel this dusty structure may contain great dangers."
```

Comment:
you *could* make this an area based mood modifier.

Reply:
tried my best, moodlet message always broke when I changed the moodlet color from 'nicegreen' to 'warning'

Reply:
try 
mood_message = "<span class='boldwarning''>This stucture fills you with a sense of foreboding. You don't know how, but it seems dangerous.</span>\n"